main.js

```diff
@@ -102,6 +255,11 @@ function GetPathColorCommandFromFill(element) {
     const fillMatch = fill.match(hexColorRegExp);
     if (fillMatch && fillMatch.length === 1)
       return ParseFillStringToPathColor(fillMatch[0]);
+
+    // Color keywords.
+    const namedColor = HexRgbForColorName(fill);
+    if (namedColor !== "")
```

Comment:
I think I would make this

`  if (namedColor)`

and maybe error out in this case? e.g. to help people realize if they had a typo in the color name. (Is there some third way of specifying a color that we don't cover yet?)